client/src/main/specs/sys/init.yaml

```diff
@@ -36,7 +36,7 @@ types:
   properties:
   - name: pgpKeys
     type: java.util.List<String>
-  - name: rootTokenPGPKey
+  - name: rootTokenPgpKey
```

Comment:
should not that be `root_token_pgp_key`:

[root_token_pgp_key](https://developer.hashicorp.com/vault/api-docs/system/init#root_token_pgp_key) (string: "") – Specifies a PGP public key used to encrypt the initial root token. The key must be base64-encoded from its original binary representation.

Reply:
Yes, that's the actual name of the final API parameter that gets generated from the YAML spec file. That's why `rootTokenPGPKey` resulted in `root_token_p_g_p_key` which is wrong.